src/test/java/com/company/sqlTest/dataProvider/VacancySqlDataProvider.java

```diff
@@ -0,0 +1,70 @@
+package com.company.sqlTest.dataProvider;
+
+import com.company.sqlTest.data.Employee;
+import com.company.sqlTest.data.URLClass;
+import com.company.sqlTest.data.UserProfile;
+import com.company.sqlTest.data.Vacancy;
+
+import java.sql.*;
+import java.util.ArrayList;
+import java.util.List;
+
+public class VacancySqlDataProvider {
+    Employee employee = new Employee();
+    Vacancy vacancy = new Vacancy();
```

Comment:
no need to store here such data - cause u return data (objects, lists, etc) from the methods